File: web3rpc/rpc-specs/code-samples/python/admin/startWS.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

adminHost = "127.0.0.1"
port = 8552
cors = ""
apis = "klay"

w3 = Web3(Web3.HTTPProvider(host))
admin_response = w3.geth.admin.start_ws(adminHost, port, cors, apis)

print(admin_response)

File: web3rpc/rpc-specs/code-samples/javascript/admin/startWS.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const host = "127.0.0.1";
  const port = 8552;
  const cors = "";
  const apis = "klay";

  provider.admin
    .startWS({ host, port, cors, apis }, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
